setup.py

```diff
@@ -1,12 +1,35 @@
-#!/usr/bin/env python
+from setuptools import find_packages
+from setuptools import setup
 
-from distutils.core import setup
-from catkin_pkg.python_setup import generate_distutils_setup
+package_name = 'xacro'
 
-d = generate_distutils_setup(
-    packages=['xacro'],
-    package_dir={'': 'src'},
-    scripts=['scripts/xacro']
+setup(
```

Comment:
I may be mistaken, but I don't believe we need both a `CMakeLists.txt` *and* a `setup.py`.
I think we can remove `setup.py` given `CMakeLists.txt` contains the following commands related to installing the Python package and the script:

    ament_python_install_package(xacro)
    install(PROGRAMS scripts/xacro DESTINATION bin)

Reply:
You are right. But, given all the effort we have put into creating the setup.py, I didn't want to delete this file again.